internal/command/views/operation.go

```diff
@@ -116,11 +116,15 @@ func (v *OperationHuman) Plan(plan *plans.Plan, schemas *terraform.Schemas) {
 
 	// Side load some data that we can't extract from the JSON plan.
 	var opts []plans.Quality
-	if !plan.CanApply() {
-		opts = append(opts, plans.NoChanges)
-	}
 	if plan.Errored {
 		opts = append(opts, plans.Errored)
+	} else if !plan.Applyable {
+		// FIXME: There might be other reasons for "non-applyable" in future,
+		// so maybe we should check plan.Changes.IsEmpty here and use a more
+		// generic fallback message if the plan doesn't seem to be empty.
+		// There are currently no other cases though, so we'll keep it
+		// simple for now.
```

Comment:
At the moment, there are none, which is why I didn't go to the trouble of complicating this. But since the idea of this was to give us the flexibility to adjust the definition of "applyable" in future -- for reasons we have not yet imagined -- this comment is intended as an aid to the hypothetical future person who is trying to do that and noticing that Terraform CLI is returning a confusing error message in whatever new situation they've introduced.

Reply:
I'm going to merge this now just because it seems like if we want to do anything in response to this comment I could do so in a separate PR, and I have some other commits building on this one that I'd like to try to finish up today. Please let me know if there's something you'd like to change about this, and I'd be happy to do it in a followup PR.

Reply:
Nope, not at all! Thanks for the clear explanation.